src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -100,6 +108,9 @@
     private OCFile mFile = null;
     private FileListListAdapter mAdapter;
 
+    private LinearLayout filesLinearLayout;
+    private LinearLayout cameraUploadLinearLayout;
```

Comment:
And cameraUploadLinearLayout as uploadFromCameraLinearLayout so that is not misunderstood as the camera uploads feature.